Source/WebCore/rendering/style/RenderStyle.cpp

```diff
@@ -2673,9 +2678,9 @@ void RenderStyle::setCustomPropertyValue(Ref<const CSSCustomPropertyValue>&& val
     auto set = [&](auto& data) {
         auto& name = value->name();
         auto* existingValue = data->customProperties->values.get(name);
-        if (existingValue && existingValue->equals(value.get()))
-            return;
-        data.access().customProperties.access().setCustomPropertyValue(name, WTFMove(value));
+    if (existingValue && existingValue->equals(value.get()))
+        return;
+    data.access().customProperties.access().setCustomPropertyValue(name, WTFMove(value));
```

Comment:
unrelated change

Reply:
Style checker was complaining, it seems someone forgot to unintend the line after the return.